usr/lib/cca_stdll/cca_stdll.h

```diff
@@ -190,4 +200,225 @@ enum cca_ktc_type {
+                    do {
+
+#define RETRY_NEW_MK_BLOB_END(tokdata, return_code, reason_code,             \
+                              blob, bloblen)                                 \
+                RETRY_NEW_MK_BLOB2_END(tokdata, return_code, reason_code,    \
+                                       blob, bloblen, NULL, 0)
+
+#define RETRY_NEW_MK_BLOB2_END(tokdata, return_code, reason_code,            \
+                               blob1, bloblen1, blob2, bloblen2)             \
+                        if ((return_code) == 8 && (reason_code) == 48) {     \
+                            TRACE_DEVEL("%s MKVP mismatch\n", __func__);     \
+                            if (!single_selected &&                          \
+                                cca_check_blob_select_single_apqn((tokdata), \
+                                                      (blob1), (bloblen1),   \
+                                                      (blob2), (bloblen2),   \
+                                                      serialno)) {           \
+                                single_selected = 1;                         \
+                                continue;                                    \
+                            }                                                \
+                        }                                                    \
+                        if (single_selected) {                               \
+                            if (cca_deselect_single_apqn((tokdata),          \
+                                                      serialno) != CKR_OK) { \
+                                TRACE_ERROR("%s Failed to de-select single " \
+                                            "APQN\n", __func__);             \
+                                (return_code) = 16;                          \
+                                (reason_code) = 336;                         \
+                            }                                                \
+                        }                                                    \
+                        break;                                               \
+                    } while (1);                                             \
+                } while (0);
+
+/*
+ * Macros to enclose any usage of CCA verbs.
+ * Obtains a READ lock on the CCA adapter lock, if a DOM-ANY configuration is
+ * used. This prevents CCA adapter usage concurrent to another thread performing
+ * Domain selection processing. Domain selection works on process scope, so
+ * it would influence all threads that currently use CCA verbs.
+ */
+#define USE_CCA_ADAPTER_START(tokdata, return_code, reason_code)             \
+                do {                                                         \
+                    if (((struct cca_private_data *)                         \
+                                      (tokdata)->private_data)->dom_any) {  \
```

Comment:
while at it, you may move the ending \ one col right

Reply:
done